web/src/types/index.ts

```diff
@@ -9,6 +9,8 @@ export interface Dandiset {
   identifier: string,
   created: string,
   modified: string,
+  // TODO most_recent_version is girderized, it should have type Version
+  most_recent_version?: any,
```

Comment:
To clarify, you're saying that we should change this type to `Version` once Girder is gone, right? Should we file an issue?

Reply:
Correct. The only reason it's any is because `girderize` isn't typed. As we remove `girderize`, I think we will run into this and retype it. I guess an issue can't hurt, though.